blaze-core/src/main/scala/org/http4s/blazecore/util/ChunkWriter.scala

```diff
@@ -41,11 +40,13 @@ private[util] object ChunkWriter {
         ByteBuffer.wrap(rr.result.getBytes(ISO_8859_1))
       } else ChunkEndBuffer
     }
-    unsafeRunAsync(f) {
-      case Right(buffer) =>
-        IO { promise.completeWith(pipe.channelWrite(buffer).map(Function.const(false))); () }
-      case Left(t) =>
-        IO { promise.failure(t); () }
+    F.start {
```

Comment:
You need to run the outer `F` to kick it off when you use `start`; as it is, this code does nothing at all anymore.